sdv/data_processing/data_processor.py

```diff
@@ -103,6 +104,112 @@ def __init__(self, metadata, enforce_rounding=True, enforce_min_max_values=True,
         if self._primary_key:
             self._keys.append(self._primary_key)
 
+    def _get_column_in_multi_column_transformer(self):
```

Comment:
we may want to come out with a nomenclature for columns that are part of multi-column transformers because these name are long and starting to clutter the code. Maybe `_get_grouped_columns` and `self.grouped_columns_to_transformers` or something like that